scripts/known_good.json

```
@@ -64,6 +64,20 @@
             "optional": [
                 "tests"
             ]
+        },
+        {
+            "name": "mimalloc",
+            "url": "https://github.com/microsoft/mimalloc.git",
+            "sub_dir": "mimalloc",
+            "build_dir": "mimalloc/build",
+            "install_dir": "mimalloc/build/install",
+            "cmake_options": [
+                "-DMI_BUILD_STATIC=ON",
+                "-DMI_BUILD_OBJECT=OFF",
+                "-DMI_BUILD_SHARED=OFF",
+                "-DMI_BUILD_TESTS=OFF"
+            ],
+            "commit": "v2.0.7"
```

Specify `build_platforms` to avoid downloads for non-win32 platforms.

```suggestion
            "commit": "v2.0.7"
            "build_platforms" : [ "windows" ]
```